x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary.tsx

```diff
@@ -36,11 +36,17 @@ interface UserRiskEntity {
 export type RiskEntity = HostRiskEntity | UserRiskEntity;
 
 const RiskSummaryComponent: React.FC<RiskEntity> = ({ risk, riskEntity, originalRisk }) => {
-  const currentRiskScore =
+  let currentRiskScore =
     riskEntity === RiskScoreEntity.host
       ? risk?.result?.[0]?.host?.risk?.calculated_level
       : risk?.result?.[0]?.user?.risk?.calculated_level;
 
+  // sometimes there is a race condition where the currentRisk is not yet available
+  // in that case, we will use the originalRisk as the currentRisk
+  if (!currentRiskScore && originalRisk) {
+    currentRiskScore = originalRisk;
```

Comment:
After our [previous investigation](https://github.com/elastic/kibana/issues/169154#issuecomment-1823015338) my [understanding](https://github.com/elastic/kibana/pull/169714#issuecomment-1823012514) was that these were failing because the new risk score was somehow not available to the UI during the test.

Am I correct in understanding that the fix here is to present the original score as the current risk score, instead? I can understand how that would cause the test to pass, but it does not seem correct/accurate, since we have explicit labels for "current" and "original" scores and the two don't seem to be interchangeable.

Reply:
Reverted in https://github.com/elastic/kibana/pull/177421/commits/f188673002316d9c8f4c4058b9e94726551af04d.  I had thought that if there was no current score then the original was equivalent to the current, but of course they are distinct.